PageSpeed_Api.php

```diff
@@ -1,128 +1,1898 @@
+				'displayValue' => $this->v( $data, array( 'lighthouseResult', 'audits', 'interactive', 'displayValue' ) ),
+			),
+			'cumulative-layout-shift'  => array(
+				'score'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'cumulative-layout-shift', 'score' ) ),
+				'displayValue' => $this->v( $data, array( 'lighthouseResult', 'audits', 'cumulative-layout-shift', 'displayValue' ) ),
+			),
+			'total-blocking-time'      => array(
+				'score'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'total-blocking-time', 'score' ) ),
+				'displayValue' => $this->v( $data, array( 'lighthouseResult', 'audits', 'total-blocking-time', 'displayValue' ) ),
+			),
+			'speed-index'              => array(
+				'score'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'speed-index', 'score' ) ),
+				'displayValue' => $this->v( $data, array( 'lighthouseResult', 'audits', 'speed-index', 'displayValue' ) ),
+			),
+			'screenshots'              => array(
+				'final' => array(
+					'title'      => $this->v( $data, array( 'lighthouseResult', 'audits', 'final-screenshot', 'title' ) ),
+					'screenshot' => $this->v( $data, array( 'lighthouseResult', 'audits', 'final-screenshot', 'details', 'data' ) ),
+				),
+				'other' => array(
+					'title'       => $this->v( $data, array( 'lighthouseResult', 'audits', 'screenshot-thumbnails', 'title' ) ),
+					'screenshots' => $this->v( $data, array( 'lighthouseResult', 'audits', 'screenshot-thumbnails', 'details', 'items' ) ),
+				),
+			),
+			'opportunities'            => array(
+				'render-blocking-resources' => array(
+					'title'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'render-blocking-resources', 'title' ) ),
+					'description'  => $this->v( $data, array( 'lighthouseResult', 'audits', 'render-blocking-resources', 'description' ) ),
+					'score'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'render-blocking-resources', 'score' ) ),
+					'displayValue' => $this->v( $data, array( 'lighthouseResult', 'audits', 'render-blocking-resources', 'displayValue' ) ),
+					'details'      => $this->v( $data, array( 'lighthouseResult', 'audits', 'render-blocking-resources', 'details', 'items' ) ),
+					'type'         => array(
+						'FCP',
+						'LCP',
+					),
+					'instruction'  => wp_kses(
+						sprintf(
+							// translators: 1 HTML a tag to W3TC minify JS section, 2 HTML a tag to W3TC minify CSS section.
+							__(
+								'W3TC can eliminate render blocking resources. Once Minified, you can deffer JS in the 
+									%1$s. Render blocking CSS can be eliminated in %2$s using the "Eliminate Render 
+									blocking CSS by moving it to HTTP body" (PRO FEATURE)',
+								'w3-total-cache'
+							),
+							'<a href="' . esc_url( '/wp-admin/admin.php?page=w3tc_minify#js' ) . '">' . esc_html__( 'Performance>Minify>JS', 'w3-total-cache' ) . '</a>',
```

Comment:
Users are able to change `wp-admin` to something else... and this will break your links.

Instead, use `admin_url( 'admin.php?page=w3tc_minify#js' )`